recipes/default.rb

```diff
@@ -16,9 +17,13 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-include_recipe 'apt::default'
-include_recipe 'ark::default'
+include_recipe 'mcrouter::_user'
+include_recipe 'mcrouter::_deps_centos' if node['platform'] == 'centos'
+include_recipe 'mcrouter::_deps_ubuntu_16.04' if node['platform'] == 'ubuntu' && node['platform_version'].to_f	== 16.04
 include_recipe 'memcached::default' if node['mcrouter']['local_memcached']
+include_recipe 'mcrouter::double_conversion'
```

Comment:
Depends on what the intention is ... do we even _want_ to allow it to be in the run-list? If not, can update it to have a leading underscore to imply it shouldn't be in a run-list.